lib/linguist/languages.yml

```diff
@@ -49,6 +49,16 @@ ABAP:
   - ".abap"
   ace_mode: abap
   language_id: 1
+ABNF:
+  type: data
+  ace_mode: text
+  extensions:
+  - ".abnf"
+  interpreters:
+  - ex_abnf
+  tm_scope: source.abnf
+  group: ABNF
```

Comment:
The `group` field here is redundant; we only use this to let one language affect a parent language's usage statistics. For example, [`GAS`](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L1326) falls under the category of `Assembly`, so we want to have it count in the usage stats of Assembly languages across GitHub.

The name is slightly misleading; a more accurate name for this field might be `parent_language`.

In short, it's logistically impossible for anything to be its own parent. ;)